modules/general/rbac-built-in-roles/main.bicep

```diff
@@ -0,0 +1,379 @@
+var roleIds = {
+  'Access Review Operator Service Role':                        '76cc9ee4-d5d3-4a45-a930-26add3d73475'
```

Comment:
Given that this module could be referenced multiple times in a deployment it will be interesting to see whether having all the roles listed risks bumping the resultant ARM template beyond the 4MB limit?

Reply:
It compiles into 30KB of ARM template, so whilst it is fairly chunky, we should be able to get away with a few of them.

What would be the alternative? Create separate modules for different categories (e.g. General, Compute, Networking, Storage, etc.)?

Reply:
No, I'm not sure there is a good alternative - being selective seems the simplest option, but then how would you choose! :-)